composer.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "lotgd/crate-www",
+    "description": "Adapts the lotgd/core engine and provides a public API for playing the game",
+    "license": "AGPL-3.0",
+    "type": "project",
+    
+    "autoload": {
+        "psr-4": {
+            "LotGD\\Crate\\WWW\\": "src/",
+            "AppBundle\\": "src/AppBundle/"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests/"
+        }
+    },
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../core"
+        }
+    ],
+    "require": {
+        "symfony/symfony": "3.1.*",
+        "doctrine/orm": "^2.5",
+        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-cache-bundle": "^1.2",
+        "symfony/swiftmailer-bundle": "^2.3",
+        "symfony/monolog-bundle": "^2.8",
+        "symfony/polyfill-apcu": "^1.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        
+        "lotgd/core": "dev-feature/buffs",
```

Comment:
I had to use this as long as my PR on lotgd/core was hanging. I'll be changing it to the normal github repository now.